### PR TITLE
Update check-website-links.yml

### PR DESCRIPTION
**Pull request by natke**
Fix typo in check links workflow

### CODE REVIEW

**Review by MaanavD — DISMISSED** [review]
Approved after my change, review again before merging though!

**Review by MaanavD — APPROVED** [review]
LGTM after discussion - checklinks is working.

**Review by sophies927 — APPROVED** [review]

**Pull request merged**
